store/tikv/scan.go

```diff
@@ -172,6 +176,11 @@ func (s *Scanner) getData(bo *Backoffer) error {
 				NotFillCache: s.snapshot.notFillCache,
 			},
 		}
+		// Reverse seek will not limit endKey.
```

Comment:
Reverse scan stops at `end_key` too. See the description here: https://github.com/pingcap/kvproto/blob/ab7debc182d9c536775b3adde7e35716f10bc0b6/proto/kvrpcpb.proto#L106-L107

Reply:
API did not add end key limitation yet. Should I add this? @disksing

Reply:
Add `EndKey` will make API unique and clear.

Reply:
fixed